src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
The plan is to target 2.13.

As @DarkDimius said, for now if you need something that’s related to the current collections you should just copy-paste it in the strawman.